views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,12 @@
+
+<div class = "beerPartial">
+    <div>
+    <a><img src={{this.image_url}} alt="beer" class = "image"></a>
+    </div>
+
+    <div>
+    <a href="/beers/{{this.id}}">{{this.name}}</a>
+    <a>{{this.description}}</a>
+    <a>{{this.tagline}}</a>
+    </div>
+</div>
```

Comment:
A small thing, but it would be nice to have the info introduced, as opposed to just pasted in there. So for instance:

<p> Brewer's tips: {{ brewers_tips}} </p>